hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/onset/WorkflowOnset.java

```diff
@@ -89,6 +89,24 @@ public OnsetResult parseAndCheck(
         return doParseAndCheck(ctx, () -> ctx.txParser().parseFrom(buffer));
     }
 
+    /**
+     * Check the validity of the provided {@link Transaction}
```

Comment:
Please elaborate on the validity part of javadocs. Will be helpful to mention checks being done in one or two sentences

Reply:
Added comments